sympy/solvers/polysys.py

```diff
@@ -641,15 +636,10 @@ def factor_system_cond(eqs: Sequence[Expr | complex], gens: Sequence[Expr] = (),
     Returns
     =======
 
-    tuple
-        A pair (systems, conditions) where:
-
-        * systems is a list of lists of (factor, conditions) pairs,
-          where factor is an expression and conditions is a list
-          of expressions that must be zero for the factor to be relevant
-
-        * conditions is a list of expressions that must be zero
-          for any solution to exist
+    list[list[Expr]]
+        A list of lists of expressions, where each sublist represents
+        an irreducible subsystem. Includes both generic solutions and
+        degenerate cases where parameters must be zero.
```

Comment:
It is not that "parameters must be zero" because e.g.:
```python
In [5]: factor_system([a-1], [x, y])
Out[5]: []

In [6]: factor_system_bool([a-1], [x, y])
Out[6]: a - 1 = 0
```
Rather it should be something like "degenerate cases requiring equality conditions on the parameters".